### PR TITLE
Bug 1389538 - Fix django-rest-framework deprecation warnings about 'rest_framework.filters'

### PR DESCRIPTION
**Pull request by ydidwania**
The deprecation warnings don't show up now. :)

### CODE REVIEW

File: treeherder/webapp/api/classifiedfailure.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 from collections import defaultdict
 
-import rest_framework_filters as filters
+import django_filters
```
Comment (edmorley) [inline treeherder/webapp/api/classifiedfailure.py]: For this and the ones below, I'd perhaps use `from django_filters import rest_framework as filters`, like so:
https://django-filter.readthedocs.io/en/develop/guide/rest_framework.html#quickstart

Comment (ydidwania) [inline treeherder/webapp/api/classifiedfailure.py]: We use `rest_framework_filters.OrderingFilter`([source](https://github.com/encode/django-rest-framework/blob/master/rest_framework/filters.py)) in treeherder/webapp/api/performance_data.py below. It is different from `django_filters.rest_framework.OrderingFilter` as can be found out from [here](https://github.com/carltongibson/django-filter/blob/develop/django_filters/filters.py#L583), which is probably why we did not get deprecation warnings for `OrderingFilter` like we did for `DjangoFilterBackend` and `FilterSet` . 

So, I suppose we can modify the `import` in all files other than that .

Comment (edmorley) [inline treeherder/webapp/api/classifiedfailure.py]: Ah yes, interesting they haven't deprecated that one.

Comment (ydidwania) [inline treeherder/webapp/api/classifiedfailure.py]: yeah, its confusing how many class share the same name in drf and django.
So should I go ahead and make the changes in other files?

Comment (edmorley) [inline treeherder/webapp/api/classifiedfailure.py]: For all the files covered by the deprecation warnings, yes please :-)

**Comment by ydidwania** [comment]
Should I rebase the commits on master ?

**Comment by edmorley** [comment]
Yes please

**Pull request merged**